scripts/do-release.sh

```diff
@@ -65,15 +65,22 @@ function clone_and_create_branch() {
     _git checkout -B "${branch}" "remotes/origin/${base_branch}"
 }
 
+# Allow skipping go commands entirely when desired by certain tests, to make the test execution faster.
+function _go() {
+    local DEBUG_PRINT=false
+    [[ -z "$SKIP_WHEN_TESTING" ]] || { echo "SKIPPING: go $*" && return 0; }
+    go "$@"
+}
+
 function _update_go_mod() {
     local target_version=${release['branch']:-devel}
     dryrun target_version="${release['version']}"
     dryrun export GONOPROXY="github.com/submariner-io/${target}"
 
-    go mod tidy -compat=1.17
-    go get "github.com/submariner-io/${target}@${target_version}"
-    go mod tidy -compat=1.17
-    go mod vendor
+    _go mod tidy -compat=1.17
+    _go get "github.com/submariner-io/${target}@${target_version}"
+    _go mod tidy -compat=1.17
+    _go mod vendor
```

Comment:
The goal here doesn’t seem to be skipping any Go command, but specifically the `update_go_mod` function. I’d rather have a specific skip for that than a generic Go skip.